crt_portal/static/js/set_referrer_on_report.js

```diff
@@ -0,0 +1,19 @@
+(function(root) {
+  referrer = document.referrer;
+  host = document.location.host;
+  if (referrer.indexOf(host) > -1) {
```

Comment:
Does this work to find if something is in a string?  I have only used it on arrays before.  Cool!